invenio/utils/deprecation.py

```diff
@@ -25,20 +25,54 @@
 completely removed in version ``A.(B+2)``.
 
 Example:
+
 - Feature X is mark in version *2.0* as deprecated. The feature will still
   work, but it will issue the silent warning ``RemovedInInvenio22Warning``
   (`PendingDeprecationWarning`).
 - Version *2.1* still contains the feature, but the warning
   ``RemovedInInvenio22Warning`` is now loud by default (`DeprecationWarning`).
 - Version *2.2* will completely remove the feature.
+
+Fast track deprecation
+----------------------
+Poorly tested, non-core and little used features for which alternatives already
+exists in Invenio may opt to use fast track deprecation. If a feature is
+fast track deprecated in version ``A.B`` it will continue to work in version
+``A.B`` but raise loud warnings. It will be completely removed in version
+``A.(B+1)``.
```

Comment:
It may be useful to expand the description a bit, to say basically that this is an exceptional process that concerns itself with features that "nobody" uses or cares about, especially in these times of v1 -> v2 transition times. Example remote debugger.  The idea is that the services shouldn't get impression that "important user facing features" they started to rely on in their LABS instances will be randomly removed... Kind of set the expectations straight.